products/zentyal.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Zentyal
+category: os
+tags: linux-distribution
+permalink: /zentyal
+releasePolicyLink: https://zentyal.com/release-policy/
+releaseDateColumn: true
+eolColumn: Development Edition Support
+eoesColumn: Commercial Support
+
+customColumns:
+-   property: ubuntuVersion
+    position: before-latest-column
+    label: Ubuntu Version
+    description: Ubuntu LTS corresponding version
+
+releases:
```

Comment:
Some automation is possible, based on the table on https://www.zentyal.com/release-policy/.

Reply:
Done in https://github.com/endoflife-date/endoflife.date/pull/5182/commits/82b42063c78596973b10e8074ba0142f425b2708. Had to drop the '.0' for most of the `releaseCycle`s.